### ISSUE TITLE
E       brownie.exceptions.VirtualMachineError: revert

### ISSUE THREAD

**Issue by upalchowdhury** [smartcontractkit/chainlink-mix#26]
I am new to the brownie framework but it seems there is fundamental issue that i can not resolve. I am testing with the basic price feed data that comes with the install. I am using using mainnet-fork from alchemy . I am getting below error while testing,

print( pricefeed.getLatestPrice())
E       brownie.exceptions.VirtualMachineError: revert

I have tested solc 0.6 and 0.8 versions and same error is reproducible in brownie console as well as python test script with brownie-config.yaml file. Please find the below file. 

Appreciate any insight. thanks

pricefeed.sol: 
==========

// SPDX-License-Identifier: MIT
pragma solidity ^0.6.6;

import "@chainlink/contracts/src/v0.6/interfaces/AggregatorV3Interface.sol";

contract PriceFeed {

    AggregatorV3Interface internal priceFeed;

    /**
     * Network: Kovan
     * Aggregator: ETH/USD
     * Address: 0x9326BFA02ADD2366b30bacB125260Af641031331
     */
    /**
     * Network: Mainnet
     * Aggregator: ETH/USD
     * Address: 0x5f4eC3Df9cbd43714FE2740f5E3616155c5b8419
     */
    constructor(address AggregatorAddress) public {
        priceFeed = AggregatorV3Interface(AggregatorAddress);
    }

    /**
     * Returns the latest price
     */
    function getLatestPrice() public view returns (int) {
        (
            uint80 roundID, 
            int price,
            uint startedAt,
            uint timeStamp,
            uint80 answeredInRound
        ) = priceFeed.latestRoundData();
        return price;
    }
}

brownie-config.yaml
===============
dependencies:
      - smartcontractkit/chainlink-brownie-contracts@1.1.1
compiler:
  solc:
    remappings:
      - '@chainlink=smartcontractkit/chainlink-brownie-contracts@1.1.1'
networks:
  mainnet-fork:
    eth_usd_price_feed: '0x5f4eC3Df9cbd43714FE2740f5E3616155c5b8419'

test.py
======
from brownie import PriceFeed, accounts, config, network
from web3 import Web3
def test_get_entrance_fee():
    account = accounts[0]
    pricefeed = PriceFeed.deploy(
        config["networks"][network.show_active()]["eth_usd_price_feed"], 
        {"from":account})
    assert lottery.getLatestPrice() !=0

**Comment by pappas999** [comment]
hi @upalchowdhury thanks for raising this. Looks like you're having an issue with a contract and script that you created as opposed to one of the ones that comes in the chainlink-mix project? Can you confirm if you get the same output if you use the PriceFeedConsumer contract and the 02_read_price_feed.py script?

**Comment by upalchowdhury** [comment]
@pappas999 Thanks for your response !!

I got it working. thanks for your hint on scripts which made me reinstall ganache cli. My initial port was 7545, changed it to 8545 might have solved the issue but not 100% sure.

I used solc 0.6.6 plus chainlink 0.6 contracts for this test.

One observation on the "02_read_price_feed.py script" the PriceFeedConsumer[-1] was throwing error but since i changed it read it from brownie-config.yaml it works.

test/pricefeedtest.py works fine except it does not return or print anything, which might be expected. anyhow thanks again for your quick response.
